src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/ReadyToRunReader.cs

```diff
@@ -379,10 +379,12 @@ internal IAssemblyMetadata R2RManifestMetadata
         }
 
         /// <summary>
-        /// Initializes the fields of the R2RHeader and R2RMethods
+        /// Minimally initializes the R2R reader.
         /// </summary>
-        /// <param name="filename">PE image</param>
-        /// <exception cref="BadImageFormatException">The Cor header flag must be ILLibrary</exception>
```

Comment:
Did you mean to remove this exception?

Reply:
@kg, I wonder if this is ready to be merged?
ILSpy would like to see if we can get this by March [here](https://github.com/icsharpcode/ILSpy/pull/3398#issuecomment-2708710610).